src/reference/config/writing-platform-configs.rst

```diff
@@ -49,277 +49,282 @@ Why Were Platforms Introduced?
 
 - Allow a compute cluster with multiple login nodes to be treated as a single
   unit.
-- Allow Cylc to elegantly handle failure of to communicate with login nodes.
-- Configure multiple platforms with the same hosts; for example you can use
-  separate platforms to submit jobs to a batch system and to background on
-  ``localhost``.
+- Allow Cylc to elegantly handle failure to communicate with login nodes.
+- Reduce the number of ssh connections required for job submission and polling.
 
 .. _Install Targets:
 
 What Are Install Targets?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Install targets represent file systems. More than one platform can use the
-same file system. It defaults to the name of the platform.
+same file system. Cylc relies on the site setup file ``global.cylc`` to determine
+which platforms share install targets. Cylc will then use this information to
+make the correct installations on remote platforms, including installation of
+files, creation of :cylc:conf:`global.cylc[install][symlink dirs]` and
+authentication keys to enable secure communication between platforms.
```

Comment:
Hemmingway says this sentence has a very high reading age. 

```suggestion
which platforms share install targets.

Cylc will setup each remote install target once. During setup it will:

  - Install workflow files
  - Symlink directories
  - Copy authentication keys (to allow secure communication)
```